tests/lib/prepare-restore.sh

```diff
@@ -370,7 +370,13 @@ prepare_project() {
     rm -rf /var/cache/snapd/aux
     case "$SPREAD_SYSTEM" in
         ubuntu-*)
-            # Ubuntu is the only system where snapd is preinstalled
+            # Ubuntu is the only system where snapd is preinstalled. However the
+            # prerm shipped with distro package is known to have been buggy, so
+            # call the one we have in the tree
+            if dpkg -l snapd ; then
+                # only when the package is actually installed
+                sh -x "${SPREAD_PATH}/debian/snapd.prerm" remove
+            fi
             distro_purge_package snapd
```

Comment:
Isn't this going to hide issues with prerm in the distro? Shouldn't we SRU a prerm fix instead?

Reply:
I can add a TODO/FIXME to drop this workaround when the release carrying the fix is published.

Reply:
I've added the TODO now.